dashboard/modules/job/job_manager.py

```diff
@@ -288,18 +284,93 @@ class JobManager:
     # Time that we will sleep while tailing logs if no new log line is
     # available.
     LOG_TAIL_SLEEP_S = 1
+    JOB_MONITOR_LOOP_PERIOD_S = 1
 
     def __init__(self):
         self._status_client = JobStatusStorageClient()
         self._log_client = JobLogStorageClient()
         self._supervisor_actor_cls = ray.remote(JobSupervisor)
 
+        self._running_jobs: Dict[str, JobStatusInfo] = dict()
+        self._recover_running_jobs()
+
+    def _recover_running_jobs(self):
+        """Recovers all running jobs from the status client.
+
+        Each will be added to self._running_jobs and reconciled.
```

Comment:
It's pretty clear from the implementation, but can you also add a comment that this restarts the monitoring loop for all running jobs?

Reply:
just did!